content/install/windows.html

```diff
@@ -4,6 +4,7 @@
 aliases:
 - /download/win
 - /downloads/win
+- /install/win
```

Comment:
Thanks for fixing this! Not sure we need a `/install/win` alias since there has never been a `/install/win` page (and the link has only existed for maybe 10 minutes)